tests/netgroup/test_netgroup.yml

```diff
@@ -116,6 +130,7 @@
       failed_when: result.changed or result.failed
 
     # netgroup and hostgroup with the same name are deprecated
+    # (check hostgroup)
```

Comment:
There's no need for tests playbook to limit lines to 80 characters as Python code. Please, join these lines in a single one.